### ISSUE TITLE
Bad Detection of filename running as a module

### ISSUE THREAD

**Issue by chrisorm** [allegroai/clearml#351]
We have a codebase that runs as a module (i.e. script needs to be run as python -m scriptdir.scriptname)
Virtual environment management is via poetry.

```python
from clearml import Task

def main():
    task = Task.init(project_name='project', task_name='task')
    task.execute_remotely()
    print("done")

if __name__=="__main":
    main()
```

If this is run as `python -m dir1.subdir.script`, the web ui shows the script path as `-m dir1.subdir`.

When trying to then enque the task, the failure is:

```Starting Task Execution:
dir1.subdir.__main__; 'dir1.subdir' is a package and cannot be directly executed
```
indicating it is either not appending the script name itself, so the script path has been wrongly detected.

Thanks

**Comment by jkhenning** [comment]
Hi @chrisorm,

Do you have anything else in that code structure? Is it part of a git repository?

**Comment by chrisorm** [comment]
yes the top level (where python is run from) is the git repo root.

**Comment by jkhenning** [comment]
And is the repository properly detected and shown in the `script` section of the experiment? Can you share the exact values there? (the specific git repository is not important, of course)

**Comment by patrickdasilva-clearml** [comment]
Hi @chrisorm , we're closing this issue as stale since it's been created too long ago. If you think the issue is still relevant, please make sure the issue persists on a recent version of ClearML (ideally the latest) before re-opening the issue. We'll gladly look at it again.
